rdf/common/src/main/scala/org/w3/banana/Prefix.scala

```diff
@@ -14,7 +14,7 @@ object Prefix {
     new PrefixBuilder(prefixName, prefixIri)
 }
 
-private [banana] class PrefixBuilder[Rdf <: RDF](
+class PrefixBuilder[Rdf <: RDF](
```

Comment:
Looks to me that this made private to enforce the use `Prefix.apply` - are you not able to use that?